src/vl-footer.js

```diff
@@ -12,6 +12,7 @@ awaitScript('vl-footer', 'https://prod.widgets.burgerprofiel.vlaanderen.be/api/v
  * @extends VlElement
  * 
  * @property {string} data-vl-identifier - De header identifier die gebruikt wordt om bij AIV de footer op te halen.
+ * @property {string} data-vl-development - Attribuut geeft aan dat de AIV ontwikkel servers gebruikt moeten worden.
```

Comment:
Is het eigenlijk niet properder om een base element te voorzien die een URL + UUID aanvaard en dan een vl-footer en een vl-footer-dev element te voorzien voor productie/dev?
Dan "vervuil" je je propere attributen interface niet.

Reply:
Had ik op zich ook voorgesteld. Langs de andere kant, als ge gewoon zolang ge geen UUID hebt iets anders in uw header zet, komt ge met dat attribuut ook nooit in aanraking he. Dus misschien is het niet zo erg ...

Reply:
Besproken dat we huidige setup behouden, het is de bedoeling dat iedereen een UUID aanvraagt voor elke omgeving (ook lokaal) en komt dus normaal gezien niet (of zeer beperkt) in aanraking met de development header.